config/suppressions.xml

```diff
@@ -160,6 +160,8 @@
             files="[\\/]src[\\/]test[\\/]java[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]PackageObjectFactoryTest.java"/>
   <suppress id="ImportControlTest"
             files="[\\/]src[\\/]test[\\/]java[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]utils[\\/]TokenUtilTest.java"/>
+  <suppress id="ImportControlTest"
```

Comment:
I added this so that I could use reflection in order to achieve 100% code coverage on the `equals` method in the `OverloadDescriptor` inner class of `OverloadMethodsDeclarationOrderCheck`. I had a lot of difficulty getting all of the checks to pass for this class due to:

1. IntelliJ inspections do not allow an inner package-level class declaration which would have allowed me to write non-reflection based unit tests for all of the branches
2. Jacoco requires 100% branch coverage on all PRs

So I think the only options were to allow usage of reflection to reach the 100% coverage, or add an exception in the `pom.xml` to allow a lesser coverage amount for this class. If there are any solutions I didn't think of, let me know.